app/views/recipes/index.html.erb

```diff
@@ -0,0 +1,35 @@
+
+<div class="container" style="width: 100%">
+  <div class="is-flex is-justify-content-end my-6">
+    <%= link_to 'Create Recipe', new_recipe_path, class: "btn btn-success cr-btn" %>
+  </div>
+  <ul style="padding: 2rem 0 0 0;">
+    <% if @recipes.size > 0%>
+    <% @recipes.each do |r| %>
```

Comment:
- [ ] your recipe page is currently does not have any recipe items to show, this because a user can not add a recipe and also there are no data in your seed file, you can try fixing this so the next reviewer can check if your project matches the wireframe, because of this, user cannot navigate to recipe detail
current recipe page 👇 
![image](https://user-images.githubusercontent.com/66097371/164499251-e111f883-9fb3-4a66-9296-f4e3135a35ec.png)